images/router/haproxy/pod.json

```diff
@@ -45,6 +45,10 @@
           {
             "name": "OPENSHIFT_INSECURE",
             "value": "${OPENSHIFT_INSECURE}"
+          },
+          {
+            "name": "DROP_SYN_DURING_RESTART",
+            "value": "0"
```

Comment:
I think you want to add this to the actual deployment config generated by the `oadm router` command. Add that environment variable to the place in the admin command where the deployment config is generated: https://github.com/openshift/origin/blob/master/pkg/cmd/admin/router/router.go#L553